webpack.web.js

```diff
@@ -24,7 +25,10 @@ module.exports = merge(common.webpackConfig, {
     new HtmlWebpackPlugin({
       template: "./src/web/index.html.ejs",
       filename: "index.html",
-      chunks: ["index"]
+      chunks: ["index"],
+      title: pkg.title,
+      description: pkg.description,
+      homepage: pkg.homepage
```

Comment:
Conversely, I could just shim all of package.json values into the options, but that felt a _wee_ bit heavy handed, so I just injected the 3 params I "cared" about.